pkg/daemon/update.go

```diff
@@ -58,6 +58,11 @@ const (
 	postConfigChangeActionReloadCrio = "reload crio"
 )
 
+var (
```

Comment:
These variable are not supposed to be updated, so this could be made const.

Reply:
- I would have made them const if they weren't paths being assembled with filepath.Join (which I assume was done as good practice for cross-platform separators, but I don't know how likely it is that this will need to be cross platform)
- If I make them constants, I can't assign to them in the test to redirect the files for testing